sdk/storage/azure-storage-blob/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
is this intend?

Reply:
Nope! This was from when I was troubleshooting and I guess I didn't delete it